scrapy/pipelines/images.py

```diff
@@ -88,6 +88,8 @@ def from_settings(cls, settings):
         s3store = cls.STORE_SCHEMES['s3']
         s3store.AWS_ACCESS_KEY_ID = settings['AWS_ACCESS_KEY_ID']
         s3store.AWS_SECRET_ACCESS_KEY = settings['AWS_SECRET_ACCESS_KEY']
+        s3store.POLICY = settings['FILES_STORE_S3_ACL']
```

Comment:
Good point! I thought the name was a reference to the S3FilesStore class, but the setting is only applied in FilesPipeline so it makes sense to create a different setting for ImagesPipeline.

I've just realised that FILES_STORE_S3_ACL is documented on the main settings page, but the other settings related to these pipelines (FILES_STORE, IMAGES_STORE, FILES_URLS_FIELD, etc.) are documented in the media pipeline section. Do you want to move FILES_STORE_S3_ACL (and the new IMAGES_STORE_S3_ACL setting) to the media pipeline section? Or do you prefer to add the new setting in the main settings page too? @redapple @kmike